### DISCUSSION TITLE
Support for new AlertRule model

### DISCUSSION THREAD

**Discussion by gxmiranda** [ansible-collections/community.grafana#293] [Q&A] [answered]
Hello!

I've currently migrated from Grafana 8.5.13 to 9.2.5, and my playbooks that use community.grafana.grafana_dashboard module don't handle alert rules defined in my json file anymore, even though the dashboards are still created.
It seems that the alert rule model and api have changed in latest grafana versions, so that creating dashboards and alerts are handled by 2 different api calls (https://grafana.com/docs/grafana/latest/developers/http_api/dashboard/ and https://grafana.com/docs/grafana/latest/developers/http_api/alerting_provisioning/#route-post-alert-rule) .
Is this a known issue to be adjusted in later versions of the ansible grafana modules or is there a way to keep using it to create dashboards and alerts?
Thanks in advance!

Best regards,
Gustavo

**Comment by rrey** [comment, marked as answer]
Hi @gxmiranda 
The API changed indeed in Grafana and the modules are not handling the new interface.
It is not planned to add support in this collection since we will migrate the existing modules to the grafana.grafana collection and leave Grafana manage evolutions.
